core/src/test/scala/unit/kafka/server/KafkaApisTest.scala

```diff
@@ -4385,15 +4451,39 @@ class KafkaApisTest {
         .setGroupId("group-3")
         .setErrorCode(Errors.INVALID_GROUP_ID.code)
 
-      val expectedOffsetFetchResponse = new OffsetFetchResponseData()
-        .setGroups(List(group1Response, group2Response, group3Response).asJava)
+      val group4Response = new OffsetFetchResponseData.OffsetFetchResponseGroup()
```

Comment:
Actually, how is this different from `group3Response`?

Reply:
group3Response and group4Response are the same. I wanted to when
 ```
group3Future.completeExceptionally(Errors.INVALID_GROUP_ID.exception)
group4Future.complete(group4Response)
```
handleOffsetFetch gives the same response.